internal/vm/remotevm/stats.go

```diff
@@ -0,0 +1,12 @@
+package remotevm
+
+import (
+	"context"
+
+	"github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/stats"
+	"github.com/Microsoft/hcsshim/internal/vm"
+)
+
+func (uvm *utilityVM) Stats(ctx context.Context) (*stats.VirtualMachineStatistics, error) {
+	return nil, vm.ErrNotSupported
+}
```

Comment:
Is this not supported or not implemented?

Reply:
Not supported at the moment.